content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -14,6 +14,31 @@ aliases:
 InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
 To find information about the latest Flux updates in InfluxDB Cloud, see [Flux release notes](/influxdb/cloud/reference/release-notes/flux/).
 
+## October 2021
+
+### API invokable scripts
+
+Use [API invokable scripts](/influxdb/cloud/api-guide/api-invocable-scripts/) to assign and execute a script through an API endpoint.
+
+### Clockface v3
+
+Refresh look and feel of InfluxDB Cloud UI. Updated icons, fonts, and layouts to improve visibility, accessibility, and user experience.
+
+### Flux update
+
+Upgrade to [Flux v0.139](/flux/v0.x/release-notes/).
+
+### Cancellation flow
+
+Update the cancellation workflow.
+
+### VS code editor
+
+Were changes made after we added support for tasks?
```

Comment:
any changes made after adding the support for tasks?

Reply:
removing to push out

@bpalani if we made changes in October in addition to support for tasks (added in September, please lmk, and we'll get those rolled in